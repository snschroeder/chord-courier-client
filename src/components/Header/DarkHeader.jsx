import React from 'react';
import './dark-header.css';

export default function DarkHeader() {
  return (
    <section className="dark-header">
      <h1 className="dark-header-h1">Chord Courier</h1>
      <h2 className="dark-header-h2">Delivering fresh chords</h2>
    </section>
  );
}
