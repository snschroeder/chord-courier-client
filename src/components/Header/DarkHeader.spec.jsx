/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';
import DarkHeader from './DarkHeader';

describe('DarkHeader component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <DarkHeader />
      </BrowserRouter>,
      div,
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the header correctly', () => {
    const wrapper = shallow(<DarkHeader />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
