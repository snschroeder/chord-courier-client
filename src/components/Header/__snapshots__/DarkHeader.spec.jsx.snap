// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkHeader component renders the header correctly 1`] = `
<section
  className="dark-header"
>
  <h1
    className="dark-header-h1"
  >
    Chord Courier
  </h1>
  <h2
    className="dark-header-h2"
  >
    Delivering fresh chords
  </h2>
</section>
`;
